### PR TITLE
[TACHYON-1428] Documented the tachyon package in preparation for the checkstyle rule

### PR DESCRIPTION
**Pull request by HorizonNet**
https://tachyon.atlassian.net/browse/TACHYON-1428

### CODE REVIEW

File: common/src/main/java/tachyon/util/ShellUtils.java [hunk 1]
```diff
@@ -145,7 +145,7 @@ protected void parseExecResult(BufferedReader lines) throws IOException {
     }
   }
 
-  /** Gets the output of the shell command. */
+  /** @return Gets the output of the shell command. */
```
Comment (aaudiber) [inline common/src/main/java/tachyon/util/ShellUtils.java]: remove "Gets"

File: common/src/test/java/tachyon/conf/TachyonConfTest.java [hunk 2]
```diff
@@ -64,8 +77,9 @@ public void beforeTests() {
     mSystemPropsTachyonConf = new TachyonConf();
   }
 
-  // test default properties
-
+  /**
+   * Tests the default properties.
```
Comment (aaudiber) [inline common/src/test/java/tachyon/conf/TachyonConfTest.java]: Tests the default common properties

**Comment by aaudiber** [comment]
Thanks for the PR! LGTM, just a couple minor comments

**Comment by HorizonNet** [comment]
@aaudiber Thanks! I updated the PR to incorporated your comments.

**Comment by aaudiber** [comment]
+1 LGTM

**Pull request merged**